### PR TITLE
[ERP-2026] Separate config components

### PR DESCRIPTION
**Pull request by NikuPAN**

### CODE REVIEW

File: src/components/tool/guide/Config/ConfigPicker.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+  const [alertDismissed, setAlertDismissed] = useState(false);
+  const onClose = () => {
+    setAlertDismissed(true);
+  };
+
+  return (
+    <>
+      <>
+        <Box>
+          {selected ? (
```
Comment (ScriptSmith) [inline src/components/tool/guide/Config/ConfigPicker.js:36]: Can we group the logic for the headings and alerts into components that are used in `ConfigSlider`, `ConfigPicker`, and `ConfigMultipleNumbers`?

**Review by ScriptSmith — CHANGES_REQUESTED** [review]

File: src/components/tool/guide/Config/ConfigPicker.js (hunk 1 above)
Comment (NikuPAN) [reply src/components/tool/guide/Config/ConfigPicker.js]: Updated

File: src/components/tool/guide/Config.js [hunk 2]
```diff
@@ -137,7 +60,7 @@ export const getCpuVendor = (config, onChange) => () => {
   const cpuVendors = ["Any", "AMD", "Intel"];
   return {
     element: (key, selected) => (
-      <ConfigGroup
+      <ConfigPicker
         key={key}
         title="CPU Vendor"
         type="picker"
```
Comment (ScriptSmith) [inline src/components/tool/guide/Config.js:66]: `type` prop no-longer needed

**Review by ScriptSmith — APPROVED** [review]

File: src/components/tool/guide/Config.js (hunk 2 above)
Comment (NikuPAN) [reply src/components/tool/guide/Config.js]: Missed this. Fixed, also fixed wrong component

**Review by ScriptSmith — APPROVED** [review]

**Pull request merged**